token/services/ttx/auditor.go

```diff
@@ -121,12 +121,13 @@ func (r *RegisterAuditorView) Call(context view.Context) (interface{}, error) {
 }
 
 type AuditingViewInitiator struct {
-	tx    *Transaction
-	local bool
+	tx                               *Transaction
+	local                            bool
+	skipAuditorSignatureVerification bool
 }
 
-func newAuditingViewInitiator(tx *Transaction, local bool) *AuditingViewInitiator {
-	return &AuditingViewInitiator{tx: tx, local: local}
+func newAuditingViewInitiator(tx *Transaction, local, skipAuditorSignatureVerification bool) *AuditingViewInitiator {
+	return &AuditingViewInitiator{tx: tx, local: local, skipAuditorSignatureVerification: skipAuditorSignatureVerification}
```

Comment:
Should we include a testing feature in this view?

Reply:
what do you mean?